README.md

```diff
@@ -81,82 +81,40 @@ Please note that this package is still under active development and the API is s
-| ------------- | -------- | ----------------------------------------------------------------------------- | ------------------------------------ | -------- | -------- |
-| `scope`       | `string` | User flow type                                                                | `['create', 'update']`               | No       | 'create' |
-| `data`        | `object` | Data passed to the API                                                        | Content depends on value of `scope`  | Yes      | none     |
-| `accessToken` | `string` | API [OAuth token](https://developers.arcadia.com/#operation/createOAuthToken) |                                      | Yes      | none     |
-| `env`         | `string` | API environment                                                               | `['local', 'sandbox', 'production']` | Yes      | none     |
-| `client`      | `string` | Name used to reference organization in app                                    |                                      | Yes      | none     |
-| `callbacks`   | `object` | Callback functions                                                            |                                      | No       | none     |
-| `uiTheme`     | `string` | UI color theme                                                                | `['light', 'dark']`                  | No       | 'light'  |
+| Name                | Type     | Description                                                                   | Options                              | Required | Default |
+| ------------------- | -------- | ----------------------------------------------------------------------------- | ------------------------------------ | -------- | ------- |
+| `accessToken`       | `string` | API [OAuth token](https://developers.arcadia.com/#operation/createOAuthToken) |                                      | Yes      | none    |
+| `env`               | `string` | API environment                                                               | `['local', 'sandbox', 'production']` | Yes      | none    |
+| `newCredentialData` | `object` | Data used to prefill new credentials                                          | Content depends on value of `scope`  | No       | none    |
+| `callbacks`         | `object` | Callback functions                                                            |                                      | No       | none    |
+| `uiTheme`           | `string` | UI color theme                                                                | `['light', 'dark']`                  | No       | 'light' |
 
-### `config.scope`
+### `config.newCredentialData`
 
-Specifies the user flow. Defaults to `create`.
+Data that would be used to pre-fill fields prior to credential submission of a new user. Only available for an access token in `create` mode.
```

Comment:
We might want to remove language around `create` and `update` mode since there's no longer a 'scope` config where you pass `create` and `update`.

Reply:
But there is a mode that you get in the utility connect token

Reply:
I tweaked it a little. Let me know what you think.